src/map.rs

```diff
@@ -178,9 +175,17 @@ impl<K, V, S> IndexMap<K, V, S> {
         }
     }
 
-    /// Create a new map with `hash_builder`
-    pub fn with_hasher(hash_builder: S) -> Self {
-        Self::with_capacity_and_hasher(0, hash_builder)
+    /// Create a new map with `hash_builder`.
+    ///
+    /// This function is `const`, so it
+    /// can be called in `static` contexts.
+    /// If you need `hash_builder` that can be made in the static context,
+    /// consider [`ahash`](https://docs.rs/ahash/0.7.4/ahash/struct.RandomState.html#method.with_seeds).
```

Comment:
it's nice to help the user but I'd like to avoid having such versioned urls - they are stale quickly - and fast-changing information - here. I would suggest we skip mentioning Ahash.

Reply:
done!